mail-mta/protonmail-bridge-bin/protonmail-bridge-bin-1.0.6.ebuild

```diff
@@ -0,0 +1,188 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit rpm
+
+DESCRIPTION="Serves ProtonMail to IMAP/SMTP clients"
+HOMEPAGE="https://protonmail.com/bridge/"
+SRC_URI="https://protonmail.com/download/${P/-bin/}-1.x86_64.rpm"
+
+RESTRICT="bindist mirror"
+
+LICENSE="MIT protonmail-bridge-EULA"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE=""
+
+DEPEND=""
+RDEPEND="
+	app-crypt/libsecret
+	dev-libs/glib:2
+	media-sound/pulseaudio
+	sys-apps/dbus
+	virtual/opengl
+	media-libs/fontconfig
+	media-libs/freetype
+	x11-libs/libICE
+	x11-libs/libSM
+	x11-libs/libX11
+	x11-libs/libxcb
+	x11-libs/libXext
+	x11-libs/libXi
+"
+BDEPEND="dev-util/patchelf"
+
+S="${WORKDIR}"
+
+QA_PREBUILT="*"
+
+src_prepare() {
+	# Bug #660366. To workaround preserving libs, set RUNPATH and rm unused libs
+	cd "${S}/usr/lib/protonmail/bridge" || die
+	patchelf --set-rpath '$ORIGIN/lib' Desktop-Bridge || die "patchelf failed"
+	patchelf --set-rpath '$ORIGIN' lib/libicui18n.so.56 || die "patchelf failed"
+	patchelf --set-rpath '$ORIGIN' lib/libicuuc.so.56 || die "patchelf failed"
+
+	rm "lib/libQt53DCore.so.5"
```

Comment:
All external (= not provided by package manager) commands need `||die`. Also, wouldn't some globs work here? Or maybe installing selectively would be shorter.

Reply:
Yes. No commands may fail for successful emerge. They shouldn't fail in normal circumstances. If they do fail, it either means the ebuild is broken (in which case we want it to die on *you*, so you'd fix it) or the system is seriously broken (in which case we want to abort ASAP).

Reply:
added missing `||die`s